src/gui/controller/ControllerCreateCase.java

```diff
@@ -0,0 +1,23 @@
+package gui.controller;
+
+import acq.ICase;
+
+/**
+ * A controller used in controllers that is a part of create case
+ */
+public abstract class ControllerCreateCase extends Controller {
+
+  /**
+   * Reference Case
+   */
+  protected ICase c;
+
+  /**
+   * Set case
+   *
+   * @param c
+   */
+  public void setC(ICase c) {
```

Comment:
Think it will be better to name this `setCase`

Reply:
It will properly be a good idea to add a `getCase` method and make c private, it will describe better what it does than using `this.c` directly in all the tab views.